.gitignore

```diff
@@ -64,6 +64,7 @@ typings/
 
 # Python cache directory
 .cache
+*.pyc
```

Comment:
This should go in a separate PR, though I am not sure if it is needed? Not seeing any .pyc files in my iodide checkout